File: NFLPool.Model/GameScore.cs
namespace NFLPool.Model
{
    public class GameScore
    {
        public int GameScoreId { get; set; }
        public Team? AwayTeam { get; set; }
        public Team? HomeTeam { get; set; }
        public int AwayScore { get; set; }
        public int HomeScore { get; set; }

        public string? WinningTeam
        {
            get
            {
                if (AwayScore > HomeScore)
                {
                    return AwayTeam?.PoolName;
                }
                else if (AwayScore < HomeScore)
                {
                    return HomeTeam?.PoolName;
                }
                else
                {
                    return null;
                }
            }
        }
    }
}

File: java/src/main/java/com/effisoft/nflpool/model/GameScore.java
package com.effisoft.nflpool.model;

public class GameScore {

    private final int gameScoreId;
    private final Team homeTeam;
    private final Team awayTeam;
    private final int homeScore;
    private final int awayScore;

    public GameScore(int gameScoreId, Team homeTeam, Team awayTeam,
                     int homeScore, int awayScore) {
        this.gameScoreId = gameScoreId;
        this.homeTeam = homeTeam;
        this.homeScore = homeScore;
        this.awayTeam = awayTeam;
        this.awayScore = awayScore;
    }

    public int getGameScoreId() {
        return gameScoreId;
    }

    public Team getHomeTeam() {
        return homeTeam;
    }

    public Team getAwayTeam() {
        return awayTeam;
    }

    public int getHomeScore() {
        return homeScore;
    }

    public int getAwayScore() {
        return awayScore;
    }

    public String getWinningTeam() {
        if(this.awayScore > this.homeScore) {
            return awayTeam.getPoolName();
        }
        if(this.awayScore < this.homeScore) {
            return homeTeam.getPoolName();
        }
        return null;
    }
}
